src/patterns/molecules/MainBanner/MainBanner.scss

```diff
@@ -0,0 +1,50 @@
+@import "../../theme/common.scss";
+
+.MainBanner{
+    align-items: center;
+    background-image: url(../../../assets/images/banner.png);
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    height: 396px;
+    padding: 20px 60px;
+    form{
+        display: flex;
+        width: 100%;
+        input{
+            width: 80%;
+        }
+        button{
+            background-color: var(--c-red);
+            border-radius: 5px;
+            color: var(--c-white);
+            font-family: $montserrat;
+            font-size: 20px;
+            width: 20%;
+        }
+    }
+    section{
```

Comment:
I'd recommend using the BEM methodology for organizing CSS selectors. Lets discuss

Reply:
implemented 🚀